### PR TITLE
Add touch tracker manipulator so users can track plots on touch devices

### PR DESCRIPTION
**Pull request by GeertvanHorrik**

### CODE REVIEW

**Comment by objorke** [comment]
there is a missing XML comment

**Comment by GeertvanHorrik** [comment]
Should be fixed.

File: Source/OxyPlot/OxyPlot.csproj [hunk 1]
```diff
@@ -60,6 +60,7 @@
     <Compile Include="Graphics\IViewCommand{T}.cs" />
     <Compile Include="Graphics\IView.cs" />
     <Compile Include="PlotController\IPlotController.cs" />
+    <Compile Include="PlotController\Manipulators\TouchTrackerManipulator.cs" />
```
Comment (objorke) [inline Source/OxyPlot/OxyPlot.csproj]: the .NET 4.0 and Silverlight projects must also be updated

Comment (GeertvanHorrik) [inline Source/OxyPlot/OxyPlot.csproj]: Done.

**Comment by objorke** [comment]
oops. change log not updated?

**Comment by GeertvanHorrik** [comment]
Wasn't even aware I needed to update a changelog. Do you want me to create a separate pr for that?

**Comment by GeertvanHorrik** [comment]
Time to start using GitReleaseNotes (once this version is out of the door) ;-)

**Comment by objorke** [comment]
GitReleaseNotes sounds interesting. Will have a look at it later!
I can take care of the change log update!

**Pull request merged**